docs/REST API/rest-api.md

```diff
@@ -8550,16 +8587,18 @@ Deletes the session specified by ID. When the user associated with the session n
 
 ## Software
 
-- [Add software](#add-software)
-- [Download software](#download-software)
-- [Delete software](#delete-software)
+- [Add software package](#add-software)
```

Comment:
Hey @marko-lisica do we need to add "package" to the name's here? I think it's simpler to have "Add software", "Download software", etc.

Reply:
Add software feature in the UI has 2 options for now - packages and App Store apps (VPP). Packages and App Store apps are subsets of **Add software** feature. Soon we'll have Fleet library which is also the type of software available for install. I think being specific with this one makes sense. So we would have `Add software package`, `Add App Store app` and `Add Fleet library app`. 
 
Download endpoint is currently used with package, but could be used with Fleet library apps as well. Delete endpoint is used with all types of software available for install. Based on this we should probably name it `Delete software available for install` and `Download software available for install`.

Wdyt?

Reply:
> Packages and App Store apps are subsets of Add software feature. Soon we'll have Fleet library which is also the type of software available for install. I think being specific with this one makes sense. So we would have Add software package, Add App Store app and Add Fleet library app.

Ah, I forgot we have two API endpoints:
- `POST /software/package`
- `POST /software/app_store_app`

We can use `POST /software/package` for Fleet library right?

Maybe we just call this "Add package" and "Add App Store app." Because we're already under "Software" and it's less words.

And then maybe we go w/ "Delete package or App Store app" and "Download package" to be more explicit (also less words)

IIRC correctly, we don't have a feature to download an App Store app.

Thoughts?

Reply:
> We can use POST /software/package for Fleet library right?

@noahtalerman I'm not sure since it is `multipart/form-data` request. For the library apps Fleet will import files into a title. But no matter if we have a new endpoint or use this one I think we can call it `Add package`